### PR TITLE
Regenerate all ANTLR parsers using version 4.11.1

### LINKED ISSUE #1615: Regenerate antlr code with 4.10.1
Update ANTLR generated files to be compatible with version 4.10.1.

### PR DESCRIPTION
**Pull request by knutwannheden**
Fixes #1615

### CODE REVIEW

File: rewrite-hcl/build.gradle.kts [hunk 1]
```diff
@@ -22,7 +22,7 @@ dependencies {
 
     compileOnly(project(":rewrite-test"))
 
-    implementation("org.antlr:antlr4:4.9.+")
+    implementation("org.antlr:antlr4:4.11.1")
```
Comment (knutwannheden) [inline rewrite-hcl/build.gradle.kts:25]: When I changed to `4.11.+` it started using 4.11.2-SNAPSHOT. I don't know enough Gradle yet to figure this out, which is why I changed it to `4.11.1`.

**Review by knutwannheden — COMMENTED** [review]
AFAICT the files in `rewrite-hcl/src/main/antlr/.antlr` don't serve any purpose, so I deleted these (generated with ANTLR 4.8).

**Comment by knutwannheden** [comment]
@traceyyoshima You reported that ANTLR 4.10 doesn't support Java 8: https://github.com/openrewrite/rewrite/pull/1616#issuecomment-1095539029. That was true for a short time after the release, but that was accidental and has been fixed in the meantime. (Do we have automated tests for this or do I manually need to check this?)

**Comment by sambsnyd** [comment]
@knutwannheden we run the java tck on jdk 8, 11, and 17. So I would expect failures out of rewrite-java-8 if this did not support jdk8

**Comment by traceyyoshima** [comment]
> @traceyyoshima You reported that ANTLR 4.10 doesn't support Java 8: [#1616 (comment)](https://github.com/openrewrite/rewrite/pull/1616#issuecomment-1095539029). That was true for a short time after the release, but that was accidental and has been fixed in the meantime. (Do we have automated tests for this or do I manually need to check this?)

Yes, we fixed the java version issue [here](https://github.com/antlr/antlr4/pull/3648).

The main issue blocking the changes was [dependencies](https://github.com/openrewrite/rewrite/issues/1615#issuecomment-1101688655) that exist on the SAAS that were stuck on an older version of ANTLR, which needed to be updated too. The dependencies have since been updated, but the work was not prioritized. We prioritized keeping the SAAS stable while the team made many heavy changes.

@sambsnyd If I remember correctly, these updates must be coordinated with the SAAS team; otherwise, version conflicts may cause outages.

**Comment by knutwannheden** [comment]
> Yes, we fixed the java version issue [here](https://github.com/antlr/antlr4/pull/3648).

Cool. I didn't know that the bug was discovered and fixed by you guys!

**Pull request merged**